### DISCUSSION TITLE
getting started with gsoc 22

### DISCUSSION THREAD

**Discussion by Sharmalm** [networkx/networkx#5279] [General]
Hello everyone,
I am Lalit mohan sharma, I am new to open source and want to contribute in GSoC 22.
I have a knowlege about python and data science.
Could you please guide me how to get started.

**Comment by MridulS** [comment]
Hello @Lalitmohansharma1! We are just in the planning phase for GSoC so we don't know things for certain right now but if you would like to know more about the development process of NetworkX and get involved with the community please go through these links on our website:

- [New Contributor FAQ](https://networkx.org/documentation/latest/developer/new_contributor_faq.html)
- [Contributor Guide](https://networkx.org/documentation/latest/developer/contribute.html)
